classes/CartRule.php

```diff
@@ -830,7 +830,7 @@ public function checkValidity(Context $context, $alreadyInCart = false, $display
             }
 
             if ($cartTotal < $minimum_amount) {
-                return (!$display_error) ? false : $this->trans('You have not reached the minimum amount required to use this voucher', [], 'Shop.Notifications.Error');
+                return (!$display_error) ? false : $this->trans('You have not reached the %s minimum amount required to use this voucher', [$context->getCurrentLocale()->formatPrice($minimum_amount, $context->currency->iso_code)], 'Shop.Notifications.Error');
```

Comment:
```suggestion
                return (!$display_error) ? false : $this->trans('The minimum amount to benefit from this promo code is %s.', [$context->getCurrentLocale()->formatPrice($minimum_amount, $context->currency->iso_code)], 'Shop.Notifications.Error');
```

This is a great idea to add the minimum amount. The error message is more useful to the user that way. I suggest using "promo code" to stay consistent with the wording used in the front office (see screenshot). I also tried to adopt a more positive tone. :-) See the Content style guide: https://zeroheight.com/80a6c5a61/v/latest/p/510e8a-components/b/2943bd 

![image](https://user-images.githubusercontent.com/70583503/150160776-3ed04f4d-2fd6-4f9d-9480-2124f4a2faf3.png)